cmd/lakectl/cmd/commit.go

```diff
@@ -77,8 +88,9 @@ func getKV(cmd *cobra.Command, name string) (map[string]string, error) {
 func init() {
 	rootCmd.AddCommand(commitCmd)
 
-	commitCmd.Flags().StringP("message", "m", "", "commit message")
-	_ = commitCmd.MarkFlagRequired("message")
+	commitCmd.Flags().BoolVar(&allowEmptyCommitMessageFlag, AllowEmptyMessageFlagName, false, "allow an empty commit message")
```

Comment:
We usually don't bind the flags to variables, but rather we just define them here in the init method, and later consume them within the command using getters.

Reply:
Is that for a reason?

Reply:
Not sure of the reason, but I think we should do it only one way or the other (unless there's a specific reason to bind the variable here)